### PR TITLE
chore: Migrate Project, Robovoyage page to Firestore

### PR DESCRIPTION
**Pull request by radhikasingh13**

### CODE REVIEW

File: pages/projects/_id.vue [hunk 1]
```diff
@@ -52,6 +52,15 @@ export default {
...
     loadedProject() {
       return this.getProjectById()(this.$route.params.id)
+    },
+    teammember() {
+      let str = ''
+      for (let i = 0; i < this.loadedProject.teamMembers.length; i++) {
+        str += this.loadedProject.teamMembers[i].member + ' , '
+      }
+      str = str.substring(0, str.length - 2)
+      console.log(str)
```
Comment (HashirGJ8842) [inline pages/projects/_id.vue]: Remove this log

**Review by HashirGJ8842 — COMMENTED** [review]
Rest all looks good to me!

**Comment by radhikasingh13** [comment]
Why?

**Comment by HashirGJ8842** [comment]
Logging is used for testing/debugging purposes, tell me a good reason why are you logging this out here, how is it beneficial to the users?

**Comment by radhikasingh13** [comment]
Sorry , so I need to remove only line 62.

**Pull request merged**